### PR TITLE
Fix TorpedoStore

### PR DESCRIPTION
**Pull request by Ilosviki**

### CODE REVIEW

**Comment by Ilosviki** [comment]
Fix the "Random", "Exception" and "Operator" problems.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package hu.bme.mit.spaceship;
-
+import java.security.SecureRandom;
```
Comment (fabiancsenge) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:2]: It's Ok, it's safe.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (fabiancsenge) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:36]: Yes, the throw keyword is needed

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (fabiancsenge) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:46]: It's okay, i think it was just a typing error.

**Review by fabiancsenge — CHANGES_REQUESTED** [review]
Please add comment to the new class memeber named generator.

**Review by fabiancsenge — APPROVED** [review]
Everything is OK

**Pull request merged**